conf/machine/raspberrypi4.conf

```diff
@@ -0,0 +1,17 @@
+#@TYPE: Machine
+#@NAME: RaspberryPi 3 Development Board (32bit)
```

Comment:
Should be RaspberryPi 4 as well as 1 line bellow.

Reply:
Thanks @shr-project . Fixed.